index.markdown

```diff
@@ -3263,7 +3263,9 @@ Designers at dxw follow these principles:
     needs their attention, and gives everyone in the team context for the work
     being done and decisions being made.
 
-#### Career progression for developers
+#### Career progression for developers in the technology team
+
+_Please note that this subsection does not apply to the GovPress team._
```

Comment:
```suggestion
#### Career progression for technologists in Delivery+
```

Reply:
It's probably worth saying that I'm not sure how much, if any of these sections apply to GovPress at all. We started to think about GovPress team principles a couple of years ago to go in alongside this stuff. I think we could probably do with a full review of the playbook to disambiguate D+ from GovPress everywhere.

Reply:
Also, I don't think we have the term Delivery+ on the company website, except in the Blog. I'm not sure whether this is something that needs changing, but I found it a little confusing.